File: 2614-maximum-count-of-positive-integer-and-negative-integer/2614-maximum-count-of-positive-integer-and-negative-integer.go
func maximumCount(nums []int) int {
    pos, neg := 0, 0
    for _, num := range nums {
        if num > 0 {
            pos++
        } else if num < 0 {
            neg++
        }
    }
    if pos > neg {
        return pos
    }
    return neg
}

File: 2614-maximum-count-of-positive-integer-and-negative-integer/2614-maximum-count-of-positive-integer-and-negative-integer.c
int maximumCount(int* nums, int numsSize) {
    int pos = 0, neg = 0;
    for (int i = 0; i < numsSize; i++) {
        if (nums[i] > 0) pos++;
        else if (nums[i] < 0) neg++;
    }
    return (pos > neg) ? pos : neg;
}